sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_patch.py

```diff
@@ -150,7 +150,9 @@ def __init__(self, credential, user=None):
         :type user: str
         """
         self._credential = credential
-        self._user = user
+        self._user = kwargs.pop("user", None)
```

Comment:
Since we are targeting Python 3 only, we should use keyword-only arguments rather than popping `kwargs`...

Reply:
yes, good point. thanks!

Reply:
updated!